.github/workflows/rust-nft-wallet-example.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Rust NFT Wallet Darwin
         run: |
           pushd rust/nft-wallet
-          dfx start --background
+          dfx start --background --host 127.0.0.1:8000
```

Comment:
Why setting the host (port) here?

Reply:
Because of the following lines in `frontend/scripts/buildIndexHtml.js`:

```
const replacement = isNetworkLocal ? "localhost:8000" : "ic0.app";
main("ROLLUP_CONNECT_SRC_DOMAIN", replacement);
```

Not sure how to replace that

Reply:
I tried to remove it, but I think something about svelte just refuses to work. Error messages are not helpful

Reply:
Tracking in this ticket: https://dfinity.atlassian.net/browse/SDK-852